cli/container.go

```diff
@@ -28,6 +28,7 @@ type container struct {
 	memorySwap       string
 	memorySwappiness int64
 	devices          []string
+	enableLxcfs      bool
 }
 
 func (c *container) config() (*types.ContainerCreateConfig, error) {
```

Comment:
I think this function is a little bit in mess, and we need to make it more clear.